connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java

```diff
@@ -526,165 +421,174 @@ private List<WorkerLoad> pickCandidateWorkerForReassignment(List<WorkerLoad> com
     }
 
     /**
-     * Task revocation is based on an rough estimation of the lower average number of tasks before
-     * and after new workers join the group. If no new workers join, no revocation takes place.
-     * Based on this estimation, tasks are revoked until the new floor average is reached for
-     * each existing worker. The revoked tasks, once assigned to the new workers will maintain
-     * a balanced load among the group.
-     *
-     * @param activeAssignments
-     * @param completeWorkerAssignment
-     * @return
+     * Revoke connectors and tasks from each worker in the cluster until no worker is running more than it would be if:
+     * <ul>
+     *     <li>The allocation of connectors and tasks across the cluster were as balanced as possible (i.e., the difference in allocation size between any two workers is at most one)</li>
+     *     <li>Any workers that left the group within the scheduled rebalance delay permanently left the group</li>
+     *     <li>All currently-configured connectors and tasks were allocated (including instances that may be revoked in this round because they are duplicated across workers)</li>
+     * </ul>
```

Comment:
Maybe this is silly question and i don't know this things.
Is there any reason to put HTML tag for comments?